3scale-labs/3scale-perf-setup
# THREESCALE-10749

# What
Create a product with user key auth

# Issue
[THREESCALE-10749 ](https://issues.redhat.com/browse/THREESCALE-10749)

# Verification

- Setup 3scale
```bash
./setup_db.sh
./setup_apimanager.sh
```
- run the setup_product.sh
```bash
chmod 771 setup_product.sh
./setup_product.sh
```
- Confirm in the ui that the product is up and has a user key

![image](https://github.com/3scale-labs/3scale-perf-setup/assets/16667688/05cf5aa3-3cab-4a4f-b0bc-eb90e78b1929)

- Test the `/nothing/<random_number>` endpoint for post and get requests e.g.
```bash
curl -X POST "https://product1-3scale-apicast-staging.apps.aucunnin.3bjq.s1.devshift.org/nothing/2030923?user_key=c00cc360c9896fe85e93a482fa724d53"
curl -X GET "https://product1-3scale-apicast-staging.apps.aucunnin.3bjq.s1.devshift.org/nothing/2030923?user_key=c00cc360c9896fe85e93a482fa724d53"
```
